release-team/role-handbooks/release-notes/README.md

```diff
@@ -87,6 +91,7 @@ https://github.com/settings/tokens
     - [X] repo:invite
 
 #### Fork the kubernetes repositories
+
 You will need to fork two repositories to you GitHub account:
 
  - [`kubernetes/sig-release`](https://github.com/kubernetes/sig-release): This is where you will push regular PRs to keep the Release Notes draft up to date.
```

Comment:
Question:

[Over here in the krel documentation](https://github.com/kubernetes/release/blob/master/docs/krel/release-notes.md#update-the-release-notes-markdown-draft) it's indicated that `krel release-notes` will create a fork of `kubernetes/sig-release` when the command runs:

> `krel release-notes` will clone k/sig-release, create a branch, write the draft markdown and then push the new branch back to your fork in GitHub.

However, a few lines down in the above:

> ... your GitHub user (the organization that owns your fork of k/sig-release)

The reason I bring this up is: **does `krel` expect a user have a fork of sig-release prior to use?

Follow-up question: does `krel` expect that the fork's repo name is `sig-release`? I participate in lots of CNCF projects – I tend to rename generic-sounding repos (i.e. `release`) to `kubernetes-release` to keep my life in github manageable.

Reply:
Not really, you can specify only your user but the tool takes a repository slug too:

```
      --fork string         the user's fork in the form org/repo. Used to submit Pull Requests for the website and draft
```